google-datacatalog-connectors-commons/src/google/datacatalog_connectors/commons/config/__init__.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/python
+#
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from .yaml_config import YamlConfig  # noqa: F401
+
+__all__ = 'YamlConfig'
```

Comment:
```suggestion
__all__ = ['YamlConfig']
```
This also prevents flak8 from complaining about F401. It's less verbose since you can remove the above `# noqa: F401`, it's up to you...